### PR TITLE
Monster map

### LINKED ISSUE #3: Media & Music
1.1 As a user, I want to have the choice to listen to music while using the car to provide a sense of integration and realism to the simulation.

Acceptance criteria : Music toggle switch playing and pausing music.

*Refers to FR 1.14

### LINKED ISSUE #5: Health Bar
2.2 As a user, I want to see the condition of my car through a ‘health bar’ on the screen.

Acceptance Criteria: When I drive into another vehicle, or an obstacle, my health points will decrease and the race will end if my health goes down to 0.

*Refers to FR 1.12

### LINKED ISSUE #81: New monster map
4.7 As a user, I want to be able to take a break from the regular racing and get my adrenaline pumping. Something like a horror game or a monster chasing me.
Acceptance criteria: A new map where a monster chases the car to showcase the implemented health bar and music.

FR: 1.21 The system shall provide a map alternative to the racing that can utilize music, sounds and a health-bar. - Interface

### PR DESCRIPTION
**Pull request by Axel-J-L**
The creation of the new monster map is now finished.

### CODE REVIEW

File: godot-resources/mods/MonsterMapMod/src/environments/monstermap/Monster.gd [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+onready var health = $HPbar
+var hp = 100
+var shake_amount = 1.0
+	
+func _ready():
+	pass
+	
+func _process(delta):
+	var lookfor = false
+	move_speed += 0.01
```
Comment (VernitaJ) [inline godot-resources/mods/MonsterMapMod/src/environments/monstermap/Monster.gd:14]: This is a great addition for the game-application that will keep the player intrigued.

File: godot-resources/mods/MonsterMapMod/src/environments/monstermap/MonsterCamera.tscn [hunk 2]
```diff
@@ -0,0 +1,27 @@
+[gd_scene load_steps=2 format=2]
+
+[ext_resource path="res://src/environments/monstermap/MonsterCamera.gd" type="Script" id=1]
+
+[node name="Spatial" type="Spatial"]
+script = ExtResource( 1 )
+
+[node name="MonsterCamera" type="ViewportContainer" parent="."]
```
Comment (VernitaJ) [inline godot-resources/mods/MonsterMapMod/src/environments/monstermap/MonsterCamera.tscn:8]: The camera angle works well and makes the monster look massive. Good choice.

**Review by drakeaxelrod — APPROVED** [review]
tested this out and it works as expected. with more time this would be a really fun project to build on

**Review by VernitaJ — COMMENTED** [review]
This is a great implementation of the concept, and if we had the concept from the start I'm sure we could have improved massively on this, but it's impressive what was achieved in one week.

**Pull request merged**